src/js/modules/comments/components/Comment.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { connect } from "react-redux";
+import injectSheet from "react-jss";
+import { Grid, Row, Col } from "react-bootstrap/lib";
+
+import { getUsers } from "../../users/selectors";
+
+const styles = {
+  Comment: {
+    fontFamily: "Minion Pro",
+    marginBottom: "18px",
+  },
+  commentInfo: {
+    marginBottom: "2px",
+  },
+  userName: {
+    fontSize: "18px",
+    fontWeight: "bold",
+    color: "#000",
+  },
+  content: {
+    color: "#000",
+    fontSize: "16px",
+    lineHeight: "1.5",
+    margin: 0,
+  },
+  bulletPoint: {
+    color: "#ccc",
+    fontSize: "12px",
+    margin: "0 4px",
+    position: "relative",
+  },
+  publishedAt: {
+    color: "#a8a8a8",
+    fontSize: "16px",
+    fontWeight: 300,
+  },
+};
+
+const Comment = ({ classes, comment, users }) => {
+  const user = users[comment.userId];
```

Comment:
This isn't bad, but why not pass the users down from the CommentThread? Like find the user there and pass it down? Or even make a selector

Reply:
This is an alternative, but it's not really that helpful. Plus, I needa make redux not slo, yo.

Reply:
lol have you experienced Redux being slow? Also I don't see why this is more optimized. In this case you have to connect users to every comment, while you can just connect users to CommentThread

Reply:
Meh it's not much faster; but this is kind of a small thing, don't you think? I don't really see the benefit.

Reply:
It's just a minor architectural thing. Makes more sense to pair up the comment with the user in a selector